functions/tips/file_system.go

```diff
@@ -25,8 +25,11 @@ import (
 )
 
 // ListFiles lists the files in the current directory.
+// Uses directory "serverless_function_source_code" as defined in the Go
+// Functions Framework Buildpack.
+// See https://github.com/GoogleCloudPlatform/buildpacks/blob/56eaad4dfe6c7bd0ecc4a175de030d2cfab9ae1c/cmd/go/functions_framework/main.go#L38.
 func ListFiles(w http.ResponseWriter, r *http.Request) {
-	files, err := ioutil.ReadDir("./")
+	files, err := ioutil.ReadDir("./serverless_function_source_code")
```

Comment:
Would you mind creating `functions/tips/serverless_function_source_code` and putting a `README.md` file inside explaining it's used for this test?

Reply:
Done